api/swagger-spec/v1beta1.json

```diff
@@ -103,18 +135,50 @@
     "description": "API at /api/v1beta1 version v1beta1",
     "operations": [
      {
-      "type": "v1beta1.EndpointsList",
+      "type": "*json.watchEvent",
```

Comment:
I 'm not sure if/what you're asking me to do here :)

On Mon, Mar 30, 2015 at 5:17 PM, Brian Grant <notifications@github.com>
wrote:

> In api/swagger-spec/v1beta1.json
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6182#discussion_r27444231>
> :
>
> > @@ -103,18 +135,50 @@
> >      "description": "API at /api/v1beta1 version v1beta1",
> >      "operations": [
> >       {
> > -      "type": "v1beta1.EndpointsList",
> > +      "type": "*json.watchEvent",
>
> Looks like fallout from @smarterclayton
> <https://github.com/smarterclayton>'s watch changes.
>
>    1. watchEvent looks like a backdoor unversioned API type (from
>    watch/json/types.go) that's not supposed to be public
>    2. I'm not sure the * syntax works -- it may well break swagger
>    validation, which currently passes
>    3. We've lost documentation of the watch payload type
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6182/files#r27444231>
> .
>